var/spack/repos/builtin/packages/py-isal/package.py

```diff
@@ -19,3 +19,8 @@ class PyIsal(PythonPackage):
 
     depends_on("python@3.7:", type=("build", "run"))
     depends_on("py-setuptools@51:", type="build")
+    depends_on("py-wheel", type="build")
+    depends_on("automake", type="build")
+    depends_on("autoconf", type="build")
+    depends_on("libtool", type="build")
+    depends_on("nasm", type="build")
```

Comment:
According to https://python-isal.readthedocs.io/en/stable/index.html#installation-with-pip it's actually possible to use an external installation of isal instead of building it in the Python package. Let's do that instead. So remove these deps and add:
```python
depends_on("libisal")

def setup_build_environment(self, env):
    # https://python-isal.readthedocs.io/en/stable/index.html#installation-with-pip
    env.set("PYTHON_ISAL_LINK_DYNAMIC", "true")

Reply:
Sounds good. I'll give it a test here.

Reply:
Looks good.